executor/brie.go

```diff
@@ -403,6 +403,7 @@ func (gs *tidbGlueSession) CreateSession(store kv.Storage) (glue.Session, error)
 
 // Execute implements glue.Session
 func (gs *tidbGlueSession) Execute(ctx context.Context, sql string) error {
+	// FIXME: br relies on a deprecated API, it may be unsafe
 	_, err := gs.se.(sqlexec.SQLExecutor).Execute(ctx, sql)
```

Comment:
Is there a reason you didn't try switching this to `ExecuteInternal`?

Reply:
It is from BR, or SIG-tools. It may be used by some code from pingcap/br. I am afraid of incompatible changes. Should tell tools guys.

Reply:
We can take a look for a future PR, lgtm